README.md

```diff
@@ -15,7 +15,7 @@ regions and process `if`, `else if`, `else` and `end if` style statements.
 # Template Samples
 
 We have created a [dotnet template samples repo](https://github.com/dotnet/dotnet-template-samples), which shows how you can use
-the Template Engine to create new templates. The samples are setup to be stand alone for specific examples. If you are in need of a sample, and it doesn't exist please [create an issue](https://github.com/dotnet/dotnet-template-samples/issues/new) in the samples repo.
+the Template Engine to create new templates. The samples are setup to be stand alone for specific examples. 
```

Comment:
Is the aim here to simplify the README file?

Reply:
@bozturkMSFT https://github.com/dotnet/dotnet-template-samples/ is dead repository. We can discuss if we want to accept such requests to dotnet/templating repository and if we want to move them to this repository and maintain them.

Reply:
Added https://github.com/dotnet/templating/issues/2522 for it